os/board/rtl8721csm/src/rtl8721csm_boot.c

```diff
@@ -85,6 +91,29 @@ extern FAR struct gpio_lowerhalf_s *amebad_gpio_lowerhalf(u32 pinname, u32 pinmo
 /************************************************************************************
  * Public Functions
  ************************************************************************************/
+#ifdef CONFIG_PRODCONFIG
+int up_prodswd(void)
```

Comment:
1. They are the same. Let's keep only one function. It could be `up_check_swd`.
2. The driver calls swd and download checks separately. But only this board can handle them using one function, up_check_swd. Let's redirect them to up_check_swd.